openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2012,7 +1991,7 @@ This section defines Credential Format Profiles for a few of the commonly used C
 
 ## W3C Verifiable Credentials
 
-Sections 6.1 and 6.2 of [@VC_DATA] define how Verifiable Credentials MAY or MAY NOT use JSON-LD. As acknowledged in Section 4.1 of [@VC_DATA], implementations can behave differently regarding processing of the `@context` property whether JSON-LD is used or not.
+Sections 6.1 and 6.2 of [@VC_DATA] define how Verifiable Credentials MAY or MAY NOT use JSON-LD [@!JSON-LD]. As acknowledged in Section 4.1 of [@VC_DATA], implementations can behave differently regarding processing of the `@context` property whether JSON-LD is used or not.
```

Comment:
If VC_DATA isn't a normative reference I'm not convinced that JSON-LD is a normative reference?

As JSON-LD is only mentioned in the context of being a normative dependency of VC_DATA, either they're both normative or they're both not?

Reply:
Agreed.  I'll make it non-normative.